apps/files/src/components/Collaborators/SharedFilesList.vue

```diff
@@ -23,7 +23,7 @@
         key="shared-with-header-cell"
         class="uk-visible@s uk-text-nowrap uk-text-meta uk-width-medium uk-text-right"
         translate-context="'People' table column"
-        v-text="$gettext('People')"
+        v-text="$gettext('Shared with')"
```

Comment:
I'm pretty sure that this just went wrong during rebasing... should be `People`

Reply:
Or was it on purpose? @LukasHirt

Reply:
On purpose

Reply:
👍